cli/jobs/nebulaml/bert-pretrain/src/environment/env.yml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+name: ACPT_Nebula
+version: 1
+build:
+  path: ./context/
+tags:
+  os: ubuntu
+  os_version: 20.04
+  hpcx: 2.10
+  mpi: openmpi
+  mpi_version: 4.1.2rc4
+  ucx: 1.12.0
+  cuda: 11.7
+  cublas: 11.10.3.66
```

Comment:
Please run `python readme.py` from cli directory to determine what all workflows are being added.

Reply:
generated with readme.py